servo-build-dependencies/android.sls

```diff
@@ -92,22 +93,13 @@ android-ndk:
       - user: servo
   cmd.run:
       # Need to filter log output to avoid hitting log limits on Travis CI
-    - name: '{{ common.servo_home }}/android/ndk/{{ android.ndk.version }}/android-ndk-{{ android.ndk.version }}-linux-x86_64.bin | grep -v Extracting'
+    - name: 'unzip {{ common.servo_home }}/android/ndk/{{ android.ndk.version }}/android-ndk-{{ android.ndk.version }}-linux-x86_64.zip'
```

Comment:
`archive.extracted` supports unzipping, so please use that instead of manually running unzip.